Composer/packages/client/.eslintrc.js

```diff
@@ -4,4 +4,15 @@ module.exports = {
     project: './tsconfig.json',
     tsconfigRootDir: __dirname,
   },
+  rules: {
+    'node/no-extraneous-import': 'off',
+  },
+  overrides: [
+    {
+      files: ['__tests__/**/*.js'],
+      rules: {
+        'node/no-missing-import': 'off',
```

Comment:
This could be removed.

Reply:
removed and added override about 'jest.d.ts' to the .eslintrc.js at the root